E621Client/Area/Tag/E621Client.cs

```diff
@@ -44,59 +27,13 @@ public partial class E621Client
-        /// <see cref="Position.After"/> will retrieve the tags 1001, 1002, 1003, etc. You should
-        /// use this method if you don't need pagination or need to avoid the limit pagination comes
-        /// with it. Moreover, this is the most efficient way to navigate through tags.
-        /// </para>
-        /// </summary>
-        /// <param name="id">
-        /// ID of the tag you want the retrieved tags the be relatively positioned to.
-        /// </param>
-        /// <param name="position">Relative position to the given tag ID.</param>
-        /// <param name="limit">The maximum number of tags returned in a single request.</param>
-        /// <param name="category">The category the retrieved tags should be in.</param>
-        /// <param name="hideEmpty">
-        /// Whether or not to include tags that have no posts associated with them.
-        /// </param>
-        /// <param name="hasWiki">Whether or not to include tags that have a page at e621's wiki.</param>
+        /// <inheritdoc cref="IE621Client.GetTagsAsync(int,Noppes.E621.Position,System.Nullable{int},System.Nullable{Noppes.E621.TagCategory},bool,System.Nullable{bool})"/>
         public Task<ICollection<Tag>> GetTagsAsync(int id, Position position, int? limit = null, TagCategory? category = null, bool hideEmpty = true, bool? hasWiki = null)
         {
             return GetTagsWithParameterAsync(null, id, position, limit, category, null, hideEmpty, hasWiki);
         }
 
-        /// <summary>
-        /// Retrieves a collection of tags based on the given parameters.
-        /// <para>
-        /// Be aware that there is a limit on the number of tags which can be retrieved with a
-        /// single call. This limit is defined at <see cref="TagsMaximumLimit"/>. Exceeding this
-        /// limit, will cause an exception to be thrown.
-        /// </para>
-        /// <para>
-        /// You can navigate through all the tags by specifying a page number. There is, however, a
-        /// limit to the value. This limit is defined in <see cref="TagsMaximumPage"/>. Exceeding
-        /// this limit, will cause an exception to be thrown.
-        /// </para>
-        /// </summary>
-        /// <param name="page">Pagination, number of the page.</param>
-        /// <param name="limit">The maximum number of tags returned in a single request.</param>
-        /// <param name="category">The category the retrieved tags should be in.</param>
-        /// <param name="order">The order in which the retrieved tags should be.</param>
-        /// <param name="hideEmpty">
-        /// Whether or not to include tags that have no posts associated with them.
-        /// </param>
-        /// <param name="hasWiki">Whether or not to include tags that have a page at e621's wiki.</param>
+        /// <inheritdoc cref="IE621Client.GetTagsAsync(int,Noppes.E621.Position,System.Nullable{int},System.Nullable{Noppes.E621.TagCategory},bool,System.Nullable{bool})"/>
```

Comment:
Seems like this `cref` is also incorrect due to it reference a non-nullable `int`

Reply:
I had some trouble changing this at first but now i've figured it out.